build-requirements.txt

```diff
@@ -5,3 +5,4 @@
 jupytext
 noteout>=0.0.3a2
 ghp-import
+yaml
```

Comment:
Jupytext already pulls in PyYAML - could we use that instead?

Reply:
We use it explicitly in our Makefile now, so probably good to make that dependency explicit as well.

Reply:
Any reason not to use PyYAML in the Makefile though?